hypothesis-python/src/hypothesis/extra/pytz.py

```diff
@@ -28,7 +28,7 @@
 import datetime as dt
 
 import pytz
-from pytz.tzfile import StaticTzInfo
+from pytz.tzinfo import StaticTzInfo  # type: ignore  # incomplete stubs
```

Comment:
Unclear if this is meant to be public or not, but it's missing from the typeshed stubs *and installing them complains with the same error*.

At minimum, mypy should not recommend installing stubs which are already installed!